README.md

```
@@ -1,337 +1,150 @@
-    main() {
-    Future.delayed(Duration(seconds: 1))
-     .then(
-     (value) => print(value.a),
-     )
-     .catchError((error)=>print(error));
-    }
+1. **Clone o repositório:**
 
+   ```bash
+   git clone https://github.com/Fernanda-Barreto/kode-start.git
+     ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)